src/util.js

```diff
@@ -8,3 +10,9 @@ export function extend(obj, props) {
 }
 
 
+/** Coerces and returns a lowercased representation of the argument.
+ *  @param {mixed} str
+ */
+export function lowercase (str) {
```

Comment:
Creating a wrapper methods almost always ends up being larger after gzip. Duplicating method/property access is free~ish.